winlocalprocessspawner/winlocalprocessspawner.py

```diff
@@ -35,17 +35,15 @@ def get_env(self):
             if key in os.environ:
                 env[key] = os.environ[key]
         return env
-
+        
     async def start(self):
         """Start the single-user server."""
         self.port = random_port()
         cmd = []
         env = self.get_env()
         token = None
-
-        cmd.append(sys.executable)
-        py_scripts_dir = os.path.join(os.path.dirname(sys.executable), 'Scripts')
-        cmd.append(os.path.join(py_scripts_dir, "jupyterhub-singleuser"))
+        

```

Comment:
I believe you have extra spaces here too?

Once this is fixed, I can merge, since the Jupyterhub commits needed for this to work were already merged

Reply:
woops, yeah, fixed both